common/git-hooks/pre-commit

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+# Called by "git commit" with no arguments.  The hook should
+# exit with non-zero status after issuing an appropriate message if
+# it wants to stop the commit.
+
+COMMAND=common/temp/node_modules/.bin/precise-commits
```

Comment:
I would also like to avoid hardcoding a path under `common/temp/node_modules` like this.